Backend/src/test/java/com/valantic/myplaylist/TrackServiceTest.java

```diff
@@ -0,0 +1,46 @@
+package com.valantic.myplaylist;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+@SpringBootTest
```

Comment:
Diese Zeile brauchen wir nur in Controller Tests. (Damit sagen wir Spring: fahre komplett Spring hoch, erstelle alle Dependencies, usw.). Das wollen wir an diese Stelle nicht:
Grund 1: Performance. das dauert sehr lange
Grund 2: Dadurch könnten wir mit @Autowired Dependencies injecten. Aber! Genau wollen wir nicht in Unit Tests.

Hier bitte löschen